tests/test_compile_models.py

```diff
@@ -106,7 +126,9 @@ def test_compile_models(
      expected_courses,
      expected_quiz_questions,
      expected_quiz_question_contents,
-     expected_quiz_multichoice_answers) = local_expected_csvs
+     expected_quiz_multichoice_answers,
+     ib_input_instances,
+     ib_pset_problems) = local_expected_csvs
 
     with open(tmp_path / "assessments.csv", 'r') as f:
         results = f.readlines()
```

Comment:
The tests are currently passing because we're not asserting on the outputs. We should add something like the following:

```python
    with open(tmp_path / "ib_input_instances.csv", 'r') as f:
        results = f.readlines()
        for i in expected_ib_input_instances:
            assert i in results

    with open(tmp_path / "ib_pset_problems.csv", 'r') as f:
        results = f.readlines()
        for i in expected_ib_pset_problems:
            assert i in results
```

I realize now that the fixtures in this repo are using line reads vs something like `DictReader` which isn't the greatest. The risk is probably not high for that causing problems, so your call as to whether you want to try and improve that aspect in this PR.